main/src/ca/uwaterloo/flix/language/phase/jvm/GenCellClasses.scala

```diff
@@ -13,15 +15,102 @@ object GenCellClasses {
     */
   def gen()(implicit root: Root, flix: Flix): Map[JvmName, JvmClass] = {
```

Comment:
Can we remove the Cell$ classes in this commit?

Reply:
Fixed!